src/olympia/addons/tasks.py

```diff
@@ -785,7 +785,7 @@ def repack_themes_for_69(addon_ids, **kw):
             else:
                 log.info('[SKIP] No need for theme repack [%s]' % addon.id)
             timer.log_interval('')
-        except (IOError, ValidationError) as exc:
+        except (IOError, ValidationError, SigningError) as exc:
```

Comment:
A test would be nice but since we're in the middle of it… not necessarily a requirement to land the code.

Reply:
yeah I was lazy.  And it'd basically just be a copy of https://github.com/mozilla/addons-server/blob/7f37eb83c24b223c12e0d781e01d99b907297de8/src/olympia/addons/tests/test_tasks.py#L462